setup.py

```diff
@@ -37,7 +35,7 @@
     keywords="fv3core",
     name="fv3core",
     packages=find_packages(include=["fv3core", "fv3core.*"]),
-    setup_requires=setup_requirements,
+    setup_requires=[],
```

Comment:
what is this change about?

Reply:
Same as when `my_dict = {}` required a type hint, mypy also wanted a type hint for line 14. Because we don't actually use setup_requirements, I did this change instead to make it clear when reading the `setup` call that we give no setup requirements.